identity-service/src/routes/authentication.js

```diff
@@ -24,28 +24,16 @@ module.exports = function (app) {
   app.get('/authentication', handleResponse(async (req, res, next) => {
     let queryParams = req.query
 
-    if (queryParams && queryParams.lookupKey && queryParams.username) {
-      const email = queryParams.username.toLowerCase()
+    if (queryParams && queryParams.lookupKey) {
       const existingUser = await models.Authentication.findOne({
         where: {
           lookupKey: queryParams.lookupKey
         }
       })
 
-      let userObj = await models.User.findOne({
```

Comment:
@SidSethi Can we have this username query param back in? Here's a scenario. Suppose hedgehog sends out two req's in parallel /authentication and /user, and user fails. No user record gets created in the db. We have the opportunity to at least log that out here to try to remedy that. The more aggressive thing would be to upsert or findOrCreate here